src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -207,6 +207,24 @@ public boolean isStudentsInSameTeam(String courseId, String student1Email, Strin
         return isStudentInTeam(courseId, student1.team, student2Email);
     }
     
+    /*
+     * update students without affecting other classes. Use with caution!!!
+     */
+    public void updateStudent(String originalEmail, StudentAttributes student) throws InvalidParametersException,
```

Comment:
Can we move this into DataMigrationForSanitizedDataInStudentAttributes? This method should never be used again so it will be better if it's not part of the StudentsLogic class to prevent future developers from getting confused.